samples/dpdk_app/sriov/sriov-pod-1.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sriov-pod-1
+  annotations:
+    k8s.v1.cni.cncf.io/networks: sriov-net-a, sriov-net-b
+spec:
+  containers:
+  - name: sriov-example
+    image: dpdk-app-centos:latest
+    imagePullPolicy: Never
+    securityContext:
+      privileged: true
+    volumeMounts:
+    - mountPath: /etc/podinfo
+      name: podinfo
+      readOnly: false
+    - mountPath: /dev/hugepages
+      name: hugepage
+    resources:
+      requests:
+        memory: 1Gi
+        #cpu: "4"
+        intel.com/intel_sriov_dpdk_a: '1'
+        intel.com/intel_sriov_dpdk_b: '1'
+      limits:
+        hugepages-1Gi: 2Gi
+        #cpu: "4"
+        intel.com/intel_sriov_dpdk_a: '1'
+        intel.com/intel_sriov_dpdk_b: '1'
+    command: ["sleep", "infinity"]
+  volumes:
+  - name: podinfo
+    downwardAPI:
+      items:
+        - path: "labels"
+          fieldRef:
+            fieldPath: metadata.labels
```

Comment:
labels  not defined in this example pod metadata, for clarity it might be better to remove it here.

Reply:
by the way, I thought you wanted to grep pci address info from env, but env not defined for this pod ...

Reply:
The "labels" will be added by Admission Controller as part of the downward API. I haven't been using Admission Controller up to this point, so I manually added. I can add a comment to the documentation at some later point highlighting that.

Device Plugin adds the PCI Address dynamically. I didn't think I needed to do anything here.

Reply:
good to know. I used env for my setup to pass in the pci info and I would love to get rid of that.

Reply:
I just had a chance to test out the env passed in by the device plugin and I saw two possibilities:
1) PCIDEVICE_INTEL_COM_INTEL_SRIOV_EAST=0000:86:0a.7
    PCIDEVICE_INTEL_COM_INTEL_SRIOV_WEST=0000:86:02.7
2) PCIDEVICE_SRIOV_COM_MLNX_SRIOV_NETDEVICE=0000:af:00.1,0000:af:00.2

One of them actually failed my script. Just want to check if you have both forms taken care.

Reply:
If you are referring to app-netutil handling the environmental variables, Zenghui coded that and I just used it. I have not tested with Mellanox, I don't have any cards. So would not surprise me if there is an issue or two with Mellanox. I should be getting a Mellanox card this week for vDPA testing, I can see if I can get SR-IOV up and running and test with it.